XenAdmin/Controls/PDSection.cs

```diff
@@ -144,6 +145,25 @@ public bool ShowCellToolTips
             set { dataGridViewEx1.ShowCellToolTips = value; }
         }
 
+        public bool ThreeColumn
+        {
+            set
+            {
+                if (value)
+                {
+                    ColumnNotes.AutoSizeMode = DataGridViewAutoSizeColumnMode.Fill;
+                    ValueColumn.AutoSizeMode = DataGridViewAutoSizeColumnMode.AllCells;
+                    ValueColumn.Width = ValueColumn.MinimumWidth;
+                }
+                else
+                {
+                    ValueColumn.AutoSizeMode = DataGridViewAutoSizeColumnMode.Fill;
+                    ColumnNotes.AutoSizeMode = DataGridViewAutoSizeColumnMode.None;
+                    ColumnNotes.Width = ColumnNotes.MinimumWidth;
+                }
+            }
+        }

```

Comment:
Right so I don't 100% understand why we need this.

Simply removing `ThreeColumn` results in what I see as a better UI:


### **before**:
![image](https://github.com/danilo-delbusso/xenadmin/assets/32554698/396fd85a-a479-42fd-b943-798ef45f7929)

### **after**:
![image](https://github.com/danilo-delbusso/xenadmin/assets/32554698/feefd0ba-f659-4ddf-8f32-3b684b5401b1)


Now in other instances of the use of `ColumnNotes` we might not want it to be anchored to the right side. However, `ThreeColumn` doesn't actually behave as expected.

I haven't quite figured out why but I don't think it's actually working. Leaving everything as is and changing the default `ValueColumn.AutoSizeMode` to `DisplayedCells` actually results in more horizontal space being taken up 👇 

![image](https://github.com/danilo-delbusso/xenadmin/assets/32554698/cd4614a3-df22-4b9f-83d2-0b33be26028b)

I don't understand why this would work since we're overriding the value.

Furthermore, if you disconnect + reconnect from a host, `pdSectionLicense?.ValueColumn?.AutoSizeMode` is back to `Fill`. However, the spacing is exactly the same. This is suggesting that the `ValueColumn.AutoSizeMode = DataGridViewAutoSizeColumnMode.AllCells;` call isn't actually making any changes.

I have tried calling `dataGridViewEx1.AutoResizeColumns();` but that doesn't seem to be helping, either. 

In general, I'd like to avoid programmatic alteration to the columns' `AutoSizeMode`s.

I would just remove `ThreeColumn` and replace it with these changes in the section's designer:

![image](https://github.com/danilo-delbusso/xenadmin/assets/32554698/e15e43c7-445b-40f0-a196-a38c7dc2d914)

Result:

![image](https://github.com/danilo-delbusso/xenadmin/assets/32554698/72f79d97-4989-4cef-9cd2-d0705b6254ec)

Reply:
I'm not very keen on the programmatic change of the autosize mode either, but with the link column set to DisplayedCells the link is pushed to the far right and in the case of the updates from CDN panel the distance was too huge (because the text in the second column was shorter than here). Let me have another loook though.